test/typecheck/continue_stmt.c

```diff
@@ -0,0 +1,13 @@
+int main() {
+  int a = 0;
+  do {
+    if (a % 2 == 0) {
+      ++a;
+      continue;
+    }
+    ++a;
+
+  } while (a < 5);
+
+  return 0;
```

Comment:
```suggestion
  int a;
  do {
    if (true) {
      a = 5;
      continue;
    }
    a = 0;

  } while (a < 5);

  return 0;
```
I suggest focusing the test on `continue` itself, utilizing a simple `true` or `false` predicate with fewer kinds of operators. This approach aims to prevent false failures due to the breakage of other features. Additionally, since we are testing the syntax rather than the semantics, it's acceptable to keep the program simple.

Reply:
Sure. Changed the predicate to `1` since we don't have `true`, `false` keywords yet.

Reply:
Oh, yes, `1`. :laughing: